syntax = "proto3";

package user;

message SignupRequest {
    string id = 1;
    string pw = 2;
    string name = 3;
}

message AuthRequest {
    string id = 1;
    string pw = 2;
}

message AuthResponse {
    string accessToken = 1;
}
